CHANGELOG.md

```diff
@@ -1,9 +1,38 @@
 # Changelog
 
 ## Current Develop Branch
+- [#9612](https://github.com/MetaMask/metamask-extension/pull/9612): Update main-quote-summary designs/styles
 
 ## 8.1.4 Tue Nov 10 2020
-- [#9612](https://github.com/MetaMask/metamask-extension/pull/9612): Update main-quote-summary designs/styles
+- [#9687](https://github.com/MetaMask/metamask-extension/pull/9687): Allow speeding up of underpriced transactions
+- [#9694](https://github.com/MetaMask/metamask-extension/pull/9694): normalize UI component font styles
+- [#9695](https://github.com/MetaMask/metamask-extension/pull/9695): normalize app component font styles
+- [#9696](https://github.com/MetaMask/metamask-extension/pull/9696): normalize deprecated itcss font styles
+- [#9697](https://github.com/MetaMask/metamask-extension/pull/9697): normalize page font styles
+- [#9740](https://github.com/MetaMask/metamask-extension/pull/9740): Standardize network settings page
+- [#9750](https://github.com/MetaMask/metamask-extension/pull/9750): Make swap arrows accessible, make swaps advanced options accessible
+- [#9766](https://github.com/MetaMask/metamask-extension/pull/9766): Use 1px borders on inputs and buttons
+- [#9767](https://github.com/MetaMask/metamask-extension/pull/9767): Remove border radius from transfer button
+- [#9764](https://github.com/MetaMask/metamask-extension/pull/9764): Update custom RPC network dropdown icons
+- [#9763](https://github.com/MetaMask/metamask-extension/pull/9763): Add confirmation for network dropdown delete action
+- [#9583](https://github.com/MetaMask/metamask-extension/pull/9583): Use `chainId` for incoming transactions controller
+- [#9748](https://github.com/MetaMask/metamask-extension/pull/9748): Autofocus input, improve accessibility of restore page
+- [#9778](https://github.com/MetaMask/metamask-extension/pull/9778): Shorten unit input with and overflow and ellipsis it
```

Comment:
Typo "with" -> "width"

Reply:
Updated!

Reply:
Updated again - I made another typo in the first update :facepalm: